DESCRIPTION

```diff
@@ -13,10 +13,9 @@ Depends: R (>= 2.10.0)
 Suggests: rgl, RUnit, shiny
 LazyLoad: true
 Imports: methods, Rcpp (>= 0.11.0), stats, graphics, zoo
-LinkingTo: Rcpp
+LinkingTo: BH, Rcpp
```

Comment:
I have to think this through.  I am not sure yet if that is good.  Maybe using the same Boost that QL was build with is better.

But this helps for win-builder.  Maybe I just fold it in and out as needed.